packages/react-native/src/primitives/PhoneNumberField/PhoneNumberField.tsx

```diff
@@ -28,7 +28,7 @@ export default function PhoneNumberField({
   );
 
   const pickerItems = useMemo(() => {
-    return dialCodes.map((dialCode) => {
+    return dialCodes?.map((dialCode) => {
```

Comment:
Should we not render the dial code picker in the scenario of no provided `dialCodes`?

Reply:
Addressed, and added test.